Scripts/Delete-Jmeter-Cluster.ps1

```diff
@@ -0,0 +1,21 @@
+param(
+    [Parameter(Mandatory = $true)]
+    [string]$namespace
+)
+
+Set-StrictMode -Version latest
+$ErrorActionPreference = "Stop"
+
+try {
+    $PathToManifestFolder = Join-Path -Path (Split-Path $(Get-Location) -Parent) -ChildPath "jmeter-kubernetes-setup"
+    
+    log "############## Removing JMeter Master and Slave pods ##############" "DarkYellow"
```

Comment:
function not defined in this script, lets switch to command here instead of using function for one line.

Reply:
Adding function. Keeping it as a standard across scripts